package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "main": "index.js",
   "scripts": {
-    "test": "renovate-config-validator ./renovate.json"
+    "test": "renovate-config-validator .github/renovate.json"
```

Comment:
We can't rename this file, since it's imported by other projects via the extends syntax and I don't think it looks in that directory by default.

eg. `"extends": ["github>balena-os/renovate-config"]`

What we should actually do is separate the self-hosted bits from the repo configuration bits into two files. Very few settings are actually related to self-hosting if you look at their docs.